docs/apm/traces/tracing-ingest.md

```diff
@@ -0,0 +1,59 @@
+---
+id: tracing-ingest
+title: Use Tracing to Troubleshoot Ingest
+sidebar_label: Use Tracing to Troubleshoot Ingest
+description: Use tracing to to search the Audit Event Index for all events that impact ingest.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The [Audit Event Index](/docs/manage/security/audit-event-index/) contains event logs in JSON format on account activities, allowing you to monitor and audit changes. You can use the `tracingIngest` source category to search the Audit Event Index for all events that impact [log ingestion](/docs/manage/ingestion-volume/log-ingestion/).
+
+## Example query
+In the following example, events logged for `_index=sumologic_system_events AND _sourceCategory=tracingIngest` indicate some impact on ingest. The example query assumes all event types are events to be informed about.
+```
+_index=sumologic_system_events AND _sourceCategory=tracingIngest
+| json field=_raw "eventName"
+| timeslice 1h
+| count by _timeslice, eventName
+| transpose row _timeslice column eventName
+```
+
+## Ingest events
+The following ingest events and accompanying reason messages appear in tracing ingest searches.
```

Comment:
I would change `tracing ingest searches` to `tracing ingest audit logs`.

Reply:
I think this needs to be expanded.
Audit logs provide more information than just the message. Important things are messages, name and amount. Name is the type of the audit log. Message is the concrete reason why the event was logged and amount is the amount of events that were reported in a single log (we aggregate them and log multiple events in one batch - one log/JSON). Especially, the amount part is important, because we had a customer asking about it some time ago.

Example audit log below. However, I don't think it is good to provide such exact example, because the format may change and we would have outdated info in the docs there, so this is just for the context of this review. Especially, that we have word beta in there!.
But I would add a *partial* example with the _details_ part below (amount, name, description) together with a description that this means that 57 spans were dropped and reported in this one audit log.

{
    "severityLevel": "Info",
    "details": {
        "amount": 57,
        "name": "SpanDropped",
        "description": "Dropped span because spans per trace limit is exceeded. Limit: 10000."
    },
    "eventType": "System",
    "accountId": "0000000000000000",
    "eventId": "bba1e1c5-42fe-4143-be5a-e82d8014837a",
    "eventName": "SpanDropped",
    "eventTime": "2023-04-06T11:26:56.234Z",
    "eventFormatVersion": "1.0 beta",
    "subsystem": "tracingIngest"
}

Reply:
Great suggestion! Done.